main.py

```diff
@@ -202,15 +220,33 @@ def clusters(methods=["GET"]):
         sw_lng = float(request.values['sw_lng'])
         start_date = datetime.date.fromtimestamp(int(request.values['start_date']))
         end_date = datetime.date.fromtimestamp(int(request.values['end_date']))
-        fatal = int(request.values['show_fatal'])
-        severe = int(request.values['show_severe'])
-        light = int(request.values['show_light'])
-        inaccurate = int(request.values['show_inaccurate'])
+        fatal = bool(request.values['show_fatal'])
```

Comment:
Clearly, there's no advantage if you extract each line to a separate function.
I was thinking of extracting this whole paragraph to a function that gets the request as a parameter and returns the variables in a tuple.
Or perhaps better yet is to use a dictionary in this case, and pass it as `kwargs` to `bounding_box_query` and to `retrieve_clusters`, since there are so many variables.